crates/owl-ms-language-server/src/workspace.rs

```diff
@@ -650,7 +1243,7 @@ impl FrameInfo {
     }
 }
 
-fn trim_string_value(value: &String) -> String {
+fn trim_string_value(value: &str) -> String {
     value
```

Comment:
The function is not complete. To extract only the pure the literal value and discarding any additional information you should consider the three cases
- `"literal"`
- `"literal"^^datatype`
- `"literal"@lang`

For a simplified presentation you could omit the anything surrounding the outer most quotes. I personally, however, would also be intereseted in seeing the datatype or language since it might have a different meaning (consider. `"handy"@en` and `"handy"@de`). @tillmo what's you opinion?

Reply:
I created #51